rack-box/scripts/install.sh

```diff
@@ -29,6 +29,7 @@ if [ "${PACKER_BUILDER_TYPE}" == "docker" ]; then
             nginx-light \
             python3 \
             python3-pip \
+            strace \
```

Comment:
Please check if we need to add strace to rack-box/http/user-data (no change may be needed if Ubuntu's standard VM image already has strace installed).